xml/chapter5/section5/subsection1.xml

```diff
@@ -226,8 +230,8 @@ function compile(exp, target, linkage) {
   produces the sequence
```

Comment:
This function only appears with two parameters ever and is reminiscent of appending two lists so I suggest we keep it. It will not introduce any variable parameters.

Reply:
I see. OK. Made the change!